### DISCUSSION TITLE
intro

### DISCUSSION THREAD

**Discussion by giscus** [widcardw/giscus-discussion#16] [Announcements]
# intro

Docs intro

https://notes.widcard.win/intro

**Comment by widcardw** [comment]
本来想着用 script 标签来直接在 Client Side 注入的，没想到 mermaid 加载的竟然还挺慢的 `¯\_(ツ)_/¯`

怎么办呢，本来换用 Astro 就是图它的快，而且 JavaScript 极少，但是这样一来又是两难了。
